### PR TITLE
Run E2E tests using dockerized monolith

### PR DESCRIPTION
**Pull request by aarongranick-okta**
## PR Checklist

- [x] Have you verified the basic functionality for this change?
- [x] Did you add tests, as appropriate, following our [Automated Test guidelines](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/2676497890/Automated+Testing+in+the+Signin+Widget)?
- [x] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?
- [ ] Did you verify the change by running [downstream monolith artifact](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/102897979/Sign-in+Widget+Development#Sign-inWidgetDevelopment-Instructionstocreateandrunthedownstreamartifact(d16t))? (Provide link to build below)
- [No] Does this PR include noticeable changes to the UI? (If yes, attach screenshots/video below)

### Issue:

- [OKTA-528463](https://oktainc.atlassian.net/browse/OKTA-528463)

### Screenshot/Video:
Screen Shot 2022-09-20 at 2 04 42 PM

### CODE REVIEW

File: scripts/e2e-monolith.sh [hunk 1]
```diff
@@ -0,0 +1,81 @@
+#!/bin/bash -x
+
+# Monolith version to test against
+export BUILDVERSION=2022.09.0-begin-289-g7f5834bbae22
+
```
Comment (vijetmahabaleshwar-okta) [inline scripts/e2e-monolith.sh:5]: Is there any way to pull down the latest monolith version? Future enhancement?

Comment (vijetmahabaleshwar-okta) [reply scripts/e2e-monolith.sh:5]: Or is this supposed to a fixed to specifically test against this version?

Comment (aarongranick-okta) [reply scripts/e2e-monolith.sh:5]: Great question. I have a task to use API to select latest/stable version automatically: OKTA-528459 part of Epic OKTA-528454
We would get that in place before we turned this on as the default E2E script

File: scripts/monolith/enable-oie.sh [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash -ex
+
+# export all environment vars from testenv
+set -o allexport
+source testenv
+set +o allexport
+
+yarn -s --cwd test/e2e ts-node ./support/monolith/enable-oie.ts
```
Comment (shuowu-okta) [inline scripts/monolith/enable-oie.sh:8]: nit: add package.json to support dir, so script can be accessed from a well named workspace

**Review by shuowu-okta — APPROVED** [review]

File: scripts/e2e-monolith.sh [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+if ! yarn build:release; then
+  echo "build failed! Exiting..."
+  exit ${TEST_FAILURE}
+fi
+finish_log_group $?
+
+# Setup E2E environment
+create_log_group "E2E Setup"
+set +e
+setup_service google-chrome-stable 89.0.4389.72-1
```
Comment (vijetmahabaleshwar-okta) [inline scripts/e2e-monolith.sh:45]: nit: I think we have later versions of chrome available. Let me find out which one is the latest

Comment (vijetmahabaleshwar-okta) [reply scripts/e2e-monolith.sh:45]: [105.0.5195.125-1](https://artifacts.aue1e.internal/artifactory/thirdparty-yum/x86_64/chrome-sel3/105.0.5195.125-1/) is the latest available. We can try running against this and see if it works.

Comment (aarongranick-okta) [reply scripts/e2e-monolith.sh:45]: Until this script replaces the default E2E script I'd like to minimize variables and keep it as close as possible to the default E2E. So if we upgrade chrome version, it should be across the board. I created a task for this: OKTA-535569

**Pull request merged**